apps/emqx/src/emqx_channel.erl

```diff
@@ -2064,7 +2064,7 @@ maybe_resume_session(#channel{
 
 maybe_shutdown(Reason, Channel = #channel{conninfo = ConnInfo}) ->
     case maps:get(expiry_interval, ConnInfo) of
-        ?UINT_MAX ->
+        ?EXPIRE_INTERVAL_INFINITE ->
```

Comment:
This seems to be an old bug: the inherited code wasn't changed.
However, it's hard to test this part naturally — we can't "wait" for such great intervals of time; and any kind mocking make test to have no sense 🤔